charts/ob-operator/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Default values for ob-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image: oceanbasedev/ob-operator:1.2.0-snapshot
```

Comment:
use the current release oceanbase/ob-operator:1.1.0

Reply:
done